datastore_entity_derives/src/lib.rs

```diff
@@ -149,7 +150,7 @@ pub fn datastore_managed(input: TokenStream) -> TokenStream {
                             None => (),
                         }
                         let datastore_property_name =
-                            property_name.unwrap_or(struct_property_name.clone());
+                            property_name.unwrap_or_else(|| struct_property_name.clone());
```

Comment:
this one was fun! `.clone` on every call, even if there was a property name =)

Reply:
ah!